pages/profile-edit/profile-edit.jsx

```diff
@@ -209,18 +209,22 @@ class ProfileEdit extends React.Component {
             </p>
             <div className="mt-4">
               <button
-                className="btn btn-primary mr-3"
+                className="btn btn-primary"
                 type="submit"
                 onClick={event => this.handleFormSubmit(event)}
                 disabled={this.state.submitting ? `disabled` : null}
               >
                 {this.state.submitting ? SUBMITTING_LABEL : PRE_SUBMIT_LABEL}
               </button>
-              {authErrorMessage}
-              {serverErrorMessage}
-              {this.state.showFormInvalidNotice && (
-                <span>Something isn't right. Check your info above.</span>
-              )}
+              <div className="d-md-inline-block mt-2 mt-md-0 ml-md-3">
```

Comment:
any reason why we we are using `md` as the breakpoint here (vs `sm` in add.jsx?)

Reply:
@mmmavis on resizing, the breakpoint for "/add" occurs in **sm** range, and "/myprofile" occurs in **md** range. For consistency, if we use **md** for both, it will work.

Reply:
Hey @AdityaSrivast thanks for your comment. Do you mind clarifying what you mean by more efficient?